src/electron/platform/android/scan-results-to-platform-data.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { PlatformData } from 'common/types/store-data/unified-data-interface';
+import { ScanResults } from './scan-results';
+
+export type ConvertScanResultsToPlatformDataDelegate = (scanResults: ScanResults) => PlatformData;
+
+export function convertScanResultsToPlatformData(scanResults: ScanResults): PlatformData | null {
```

Comment:
I felt that it made the contract of the function more apparent, so I have a fairly strong preference for keeping it, but technically no because we aren't using strict null checks

Reply:
so if we do this.... this will be one of the only places where we use `| null` (if not the only one)... whoever read this code without the context of this comments will wonder why do we have it here and not on other places... why this odd difference. So please don't get me wrong, I would love to have strict null checks on our projects but we don't and we just explicitly check for null when needed... or suffer from it when a related bug makes it to production.

Reply:
That's a fair point, and the combination of use case sites and tests do a fair job of clarifying the nullability expectations already. Will remove it.